Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -473,7 +473,7 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
         // Only consider applying authentication credentials if this is actually a redirect and the redirect
         // URL didn't include credentials of its own.
         if (m_user.isEmpty() && m_password.isEmpty() && !redirectResponse.isNull()) {
-            auto credential = m_session->networkStorageSession() ? m_session->networkStorageSession()->credentialStorage().get(m_partition, request.url()) : WebCore::Credential();
+            auto credential = m_session->networkStorageSession() ? m_session->checkedNetworkStorageSession()->credentialStorage().get(m_partition, request.url()) : WebCore::Credential();
```

Comment:
Are these changes based on static analysis? `credentialStorage()` is trivial for example so this change doesn't look correct to me. I haven't checked the rest but I have concerns if this is not based on static analysis results.

Reply:
Sorry, I saw two warnings on this line and did not check carefully the column numbers, so in the end those were unrelated warnings, I ended that change.